CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+* PayPal
```

Comment:
Let's still call these out in a breaking changes section since this is technically a breaking change too

Reply:
Good catch! i just added the breaking changes section.